main.go

```diff
@@ -53,22 +54,24 @@ More info: https://github.com/codercom/sshcode
 
 	const codeServerPath = "/tmp/codessh-code-server"
 
-	// Downloads the latest code-server and allows it to be executed.
-	sshCmd := exec.Command("ssh",
-		"-tt",
-		host,
-		`/bin/bash -c 'set -euxo pipefail || exit 1
-wget -q https://codesrv-ci.cdr.sh/latest-linux -O `+codeServerPath+`
+	downloadScript := `/bin/bash -c 'set -euxo pipefail || exit 1
+wget -q https://codesrv-ci.cdr.sh/latest-linux -O ` + codeServerPath + `
```

Comment:
why two wget?

Reply:
No idea how that got there.. That script is completely broken. We're going to want some tests

Reply:
I think its a good idea we http.Get on our side and then just write the binary to the remote side if it has changed. Would be nice and easy to test as well versus bash scripts.

Reply:
The problem is the bin is 90MB.